_pages/en/strategy-version-control-systems.md

```diff
@@ -0,0 +1,366 @@
+---
+layout: default
+title: Version Control Systems Strategy
+ref: version-control-systems
+lang: en
+status: posted
+sections: Strategies
+version: 0.01
+date: "2020-11-05"
+permalink: /strategy-version-control-systems.html
+---
+
+<!-- markdownlint-disable MD033 -->
+<!-- the below cSpell statement says to ignore any text between HTML tags. E.g. it will ignore "th rowspan='2'" in this string: <th rowspan='2'> -->
+<!-- cSpell:ignoreRegExp /\<[^\>]+\>/ -->
+
+<style>
+table, th, td {
+  border: 1px solid black;
+}
+
+th {
+  background-color: #ccccff;
+}
+</style>
+
+<div class="alert alert-warning">
+  <strong>This is a DRAFT strategy</strong>.
+</div>
+
+<!-- markdownlint-disable MD001 -->
+##### Table of Content <!-- omit in toc -->
```

Comment:
Recommend adding an Appendix - Traceability Matrix

IITB Way Forward is mentioned but unclear how this strategy supports it.
PwC Independent Study would also be a good Traceability Matrix to add
Information Management Policy (TB Directive on Service and Digital now, but ESDC has an Information Management Strategy too)
GC Architecture Standards
If there's something in the DOSP would be a bonus

Reply:
Very relevant, thanks! Added that new section to the document as well.